### PR TITLE
add Catalog, Cart, OrderSummary, FaceRegistration component, and add it to FacePaymentPage

### PR DESCRIPTION
**Pull request by gregoriusjimmy**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

- Face Payment page has no registration form
- Face Payment page has no stepper component
- There is no Catalog, Cart, OrderSummary component
- Button has no rect style

## What is the new behavior?

- Face Payment page has registration form with webcam photo.
- Face Payment page has a stepper component.
- Add Catalog, Cart, OderSummary component,
- Add rect as Button's attribute

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
image
image
image
image
image
image
image

### CODE REVIEW

File: frontend/app/api/analyticsAPI.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import axios, { AxiosError } from 'axios'
+import { SESSION_ID_ERROR } from '../constants/message'
 import {
   AnalyticsError,
   AnalyticsResponse,
   ServiceBySlugResponse
 } from '../types/responses'
 
 const ERROR_MESSAGE = 'Something wrong has happened'
```
Comment (rizkybaihaqy) [inline frontend/app/api/analyticsAPI.ts:9]: ini gak sekalian aja di pindah ke contant?

Comment (gregoriusjimmy) [reply frontend/app/api/analyticsAPI.ts:9]: mksdnya ki?

Comment (williammfu) [reply frontend/app/api/analyticsAPI.ts:9]: emang udah di constant kan?

Comment (rizkybaihaqy) [reply frontend/app/api/analyticsAPI.ts:9]: file constant di folder constant

Comment (gregoriusjimmy) [reply frontend/app/api/analyticsAPI.ts:9]: oh nanti aja itu kita refactor besok ki

Comment (rizkybaihaqy) [reply frontend/app/api/analyticsAPI.ts:9]: okok

**Review by rizkybaihaqy — APPROVED** [review]

**Review by williammfu — APPROVED** [review]

**Pull request merged**